src/main/java/org/folio/consortia/service/impl/SharingInstanceServiceImpl.java

```diff
@@ -145,7 +144,7 @@ public SharingInstanceCollection getSharingInstances(UUID consortiumId, UUID ins
     consortiumService.checkConsortiumExistsOrThrow(consortiumId);
     var specification = constructSpecification(instanceIdentifier, sourceTenantId, targetTenantId, status);
 
-    var sharingInstancePage = sharingInstanceRepository.findAll(specification, PageRequest.of(offset, limit));
+    var sharingInstancePage = sharingInstanceRepository.findAll(specification, OffsetRequest.of(offset, limit));
```

Comment:
also we need to fix the second issue described in the ticket when the tenants array is completely skipped instead of empty array. I assume it would be enough to delete just this line:
https://github.com/folio-org/mod-consortia/blob/master/src/main/java/org/folio/consortia/config/AppConfig.java#L50

Reply:
First I thought, it is don't affect, since there is contanst 0,1 values. Now I think I agree that all places should be consistent, Changed, thanks